sdk/identity/azure-identity/azure/identity/aio/_credentials/managed_identity.py

```diff
@@ -110,17 +107,12 @@ def __init__(self, config: "Optional[Configuration]" = None, **kwargs: "Any") ->
 
 
 class MsiCredential(_AsyncManagedIdentityBase):
-    """Authenticates via the MSI endpoint in an App Service or Cloud Shell environment.
+    """Authenticates via the MSI endpoint in an App Service or Cloud Shell environment."""
```

Comment:
This should probably be a more broad statement since we want to support all hosting environments which support Managed Identity.

Reply:
This credential won't be in in published documentation, and my intent with this description of it is to claim support only for tested scenarios I expect to work. However, `ManagedIdentityCredential` will be in published docs and I wrote its blurb with the same intent. I can broaden that blurb to e.g. "authenticates with an Azure managed identity". Given the ongoing proliferation of incompatible managed identity interfaces, I'm reluctant to claim support for all of them, but I suppose any lack of support on our part would be a bug so I may as well. Will do.